website/static/js/osfLanguage.js

```diff
@@ -80,5 +80,10 @@ module.exports = {
         dataSendError: 'Error sending data to the server: check that all fields are valid, or contact ' + SUPPORT_LINK + ' if the problem persists.',
         creationSuccess: 'Successfully registered new application',
         dataUpdated: 'Application data updated'
+    },
+    projectSettings: {
+        updateSuccessMessage: 'Category, title, and description updated successfully',
+        updateErrorMessage: 'Error updating project settings',
```

Comment:
@sloria Should I just set it so that if it's a Bad Request to return "Invalid input." 
and otherwise to display REFRESH_OR_SUPPORT as Andy said?